src/app.js

```diff
@@ -1,9 +1,14 @@
 const express = require('express');
 const path = require('path');
 const app = express();
-
+const bodyParser = require('body-parser');
 const PORT = process.env.PORT || 3000;
 
+app.post('/api/forms/contact', bodyParser.urlencoded(), function(req, res) {
```

Comment:
When you run this, you probably see a warning in the console about the body-parser default `extended` option being deprecated. That message is referring to [this option](https://github.com/expressjs/body-parser#extended) for `bodyParser.urlencoded()`. To address that warning, you can pass an options object as the first argument to `bodyParser.urlencoded()` like this:

```javascript
bodyParser.urlencoded({ extended: true })
```